src/CsvHelper.Excel/ExcelParser.cs

```diff
@@ -83,8 +83,9 @@ public class ExcelParser : IParser
         /// <param name="culture">The culture.</param>
         /// <param name="leaveOpen"><c>true</c> to leave the <see cref="TextWriter"/> open after the <see cref="ExcelParser"/> object is disposed, otherwise <c>false</c>.</param>
         public ExcelParser(Stream stream, string sheetName, CultureInfo culture, bool leaveOpen = false) : this(stream,
-            sheetName, new CsvConfiguration(culture) {LeaveOpen= leaveOpen})
+            sheetName, new CsvConfiguration(culture))
         {
+            _leaveOpen = leaveOpen;
```

Comment:
This constructor should just pass `leaveOpen` to the "master" constructor to be consistent